### DISCUSSION TITLE
倍速播放视频时声音卡顿

### DISCUSSION THREAD

**Discussion by AlvinYuXing** [ywmoyue/biliuwp-lite#948] [bug]
### Bug 描述

播放视频是选择倍速播放或者长按倍速播放，视频声音会出现卡顿

### 操作步骤

```plain text
1.打开应用
2.播放视频
3.选择倍速，或者长按→，或长按屏幕倍速播放视频
4.视频声音会出现短暂停顿
```

### 期望结果

开始倍数播放视频声音无停顿

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 11 23H2

### 设备类型

笔记本

### 设备架构

Arm64

### 应用架构

Arm64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
https://github.com/ywmoyue/biliuwp-lite/issues/918

遗留问题，暂时没什么好的解决方法
